cmd/av/init.go

```diff
@@ -52,7 +52,7 @@ var initCmd = &cobra.Command{
 
 		cu.Cancel()
 		if err := tx.Commit(); err != nil {
-			return err
+			return fmt.Errorf("Could not get write to local DB: %w", err)
```

Comment:
No need to wrap these errors. Generally we only wrap where we can provide additional context, but in these cases, we're not providing any additional context (the reason for the failure to write should be provided by the error returned by `tx.Commit`), so don't wrap them.

Reply:
I think all of the error wraps you've added in this file are unnecessary, can you just remove them all?